### ISSUE TITLE
UX problem: Login Failed / resource temporarily unavailable

### ISSUE THREAD

**Issue by ligi** [status-im/status-desktop#2362]
I played around with the recent version of status-desktop. Then I closed the app (by clicking the X on the top right)
And then when trying to open the app again I am seeing this:

![image](https://user-images.githubusercontent.com/111600/115755856-135db280-a39e-11eb-9d04-a6fcd21caef5.png)

and this in the console:

```
Network Cache Dir:  "/home/ligi/.config/Status/tmp/netcache"
DBG 2021-04-22 18:54:53+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:53+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:53+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:54+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:54+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:54+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:54+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:54+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:54+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:54+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:54+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:54+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:54+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:54+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:54+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:54+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:54+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:54+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:55+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:55+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:55+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:55+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:55+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:55+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
DBG 2021-04-22 18:54:55+02:00 Login requested                            tid=69943 file=accounts.nim:214 user="Calm Incomplete Galapagospenguin"
ERR 2021-04-22 18:54:55+02:00 node.crashed                               topics="signals" tid=69943 file=core.nim:66 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable\"}}"
ERR 2021-04-22 18:54:55+02:00 node.login                                 topics="signals" tid=69943 file=core.nim:63 error="failed to create database at /home/ligi/.config/Status/data/ethereum/mainnet: resource temporarily unavailable"
```

So the app was not really closed as it seems. After seeing this in the console I could also see the icon in the status-bar:

![image](https://user-images.githubusercontent.com/111600/115757004-a0087080-a39e-11eb-9b71-1a75e7208b36.png)

but on this monitor it was really only visible a bit under a certain angle.
So I think: 
 * this icon should have better visibility
 * the error message in the UI should be improved (really thought this is a wrong password and tried a lot there before looking in the console - other users might be totally thrown off by this)
 * ideally the behavior of opening a second instance should be removed or made optional

**Comment by iurimatias** [comment]
hey @ligi , by any chance did you already have another version of status desktop running at the same time?

**Comment by iurimatias** [comment]
nevermind, this is exactly what you were saying, the app was closed but turns out it was still running

**Comment by iurimatias** [comment]
Hi @ligi thank you for reporting these issues, they are 3 issues in total and this is the follow up:

> `this icon should have better visibility`

this should now have been fixed by pr #2823

> `the error message in the UI should be improved (really thought this is a wrong password and tried a lot there before looking in the console - other users might be totally thrown off by this)`

agree, this is being tracked in issue #2843 

> ideally the behaviour of opening a second instance should be removed or made optional

There are two things that a different instance needs to do to not conflict, the port configuration (which is currently saved on account creation), and the data directory, these should be set by env vars `STATUS_PORT=30311` and `NIM_STATUS_CLIENT_DEV=f`

However a user can't be expected to guess this and we should document this, when we do it will be found in the "Advanced Options" of the user guide here https://hackmd.io/@status-desktop/B1naRjxh_/https%3A%2F%2Fhackmd.io%2F%40status-desktop%2FrkUZ6fWnO

Thank you for bringing this up, since these issues are tracked or fixed I'll close this issue
